cmd/jaeger/docs/migration/badger-metrics.md

```diff
@@ -0,0 +1,106 @@
+### Common Metrics
+
+| Metric Name | Inner Parameters |
+|-------------|------------------|
+| jaeger_badger_compaction_current_num_lsm |  |
```

Comment:
the storage-specific files will have large overlap with all-in-one, I recommend filtering out only the storage-related metrics.